docs/source/noise/index.rst

```diff
@@ -0,0 +1,58 @@
+.. _noise_main:
+
+======
+ Noise
+======
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: none
+
+In order to have a realistic challenge with entity resolution, it is essential
+to add noise to the data. Pseudopeople can add two broad categories of noise to
+the datasets it generates:
+
+#. **Row-based noise:** Errors in the inclusion or exclusion of entire rows of
+   data, such as duplication or omission
+#. **Column-based noise:** Errors in individual data entry, such as miswriting
+   or incorrectly selecting responses
+
+Pseudopeople applies the different types of row-based and column-based noise in
+the following order to mimic the data generation process by which a real dataset
+might be corrupted. The final column of the table, "Config Key," is the name of
+the noise type in the configuration dictionary used to generate the noised
+datasets.
+
+.. list-table:: Types of noise in order of application
+  :widths: 1 2 5 1
+  :header-rows: 1
+
+  * - Order
+    - Noise Type
+    - Description
+    - Config Key
+  * - ---
+    - **Row-based noise**
+    - ---
+    - ``row_noise``
+  * - 1
+    - Omit row
+    - Omit an entire randomly selected row of data
+    - ``omit_row``
+  * - ---
+    - **Column-based noise**
+    - ---
+    - ``column_noise``
+  * - 2
+    - Leave blank
+    - Leave a field blank, such as not writing your name on the designated line
```

Comment:
The grammar here is a bit awkward; I could see:

```suggestion
    - Leaving a field blank, such as not writing your name on the designated line
```

or

```suggestion
    - Leave a field blank, for example by not writing your name on the designated line
```

Also, I wonder if the use of "you" here will feel strange as we have to describe more complex noise functions. I like it because it feels unpretentious, but using "a simulant" instead would be more precise.

Reply:
I rearranged the table and changed the wording.

Reply:
I kept the examples using "you." I'll leave it to @pletale to decide whether this starts sounding weird as they add more noise types to the table.